routes/AuthRoute.js

```diff
@@ -0,0 +1,64 @@
+const router = require('express').Router()
+const Joi = require('@hapi/joi')
+const User = require('../models/User')
+
+const createSchema = Joi.object({
+    name: Joi.string()
+        .min(6)
+        .required(),
+    email: Joi.string()
+        .required()
+        .email(),
+    password: Joi.string()
+        .required()
+        
+})
+const loginSchema = Joi.object({
+    email: Joi.string()
+        .required()
+        .email(),
+    password: Joi.string()
+        .required()
+        
+})
+
+router.post('/create', async (request, response) => {
```

Comment:
name the route with this kind of naming '/api/v1/auth/create' instead of current one